demos/database.php

```diff
@@ -14,7 +14,32 @@
     ]))->addMoreInfo('PDO error', $e->getMessage());
 }
 
-$app->db = $db;
+
+if (!trait_exists('ModelLockTrait')) {
```

Comment:
Why if trait exists trait?

Reply:
Not sure what you mean here...

Reply:
Remove `if (!trait_exists('ModelLockTrait')) {`

Reply:
done